boot/zephyr/Kconfig

```diff
@@ -71,38 +71,38 @@ config BOOT_SIGNATURE_KEY_FILE
 config MBEDTLS_CFG_FILE
 	default "mcuboot-mbedtls-cfg.h"
 
-config BOOT_VALIDATE_SLOT0
-	bool "Validate image slot 0 on every boot"
+config BOOT_VALIDATE_PRIMARY_SLOT
```

Comment:
Although Zephyr defaults to enabled changing this config parameter's name enables it for people who have previously disabled it.

Reply:
I don't think we should change the name of configuration options.